docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -360,6 +360,8 @@ each of the bash functions below.
 
    If you need them in development:
 
+   ! DO NOT RUN THIS IN E2E, INT! !
+
    ```bash
    az keyvault certificate import \
```

Comment:
is `certificate` correct here? We download these from `secret`: https://github.com/Azure/ARO-RP/blob/b26372f658ecc4c340c339e324a7ad0512b397d8/pkg/deploy/generator/resources.go#L1113 
or is it different between prod and int? @mjudeikis 
(I know it is not part of your PR, but I am curious)

Reply:
Prod it is generated by keyvault (signed by msft keyvault integration). 
In int we started doing the same, so this regressed in certificate rotation.